documentation/src/main/asciidoc/stories/assembly_transactions.adoc

```diff
@@ -0,0 +1,16 @@
+[id='transactions']
+:context: transactions
+= Configuring locking and transactions
+
+Data that resides on a distributed system is vulnerable to errors that can arise from temporary network outages, system failures, or just simple human error.
+These external factors are uncontrollable but can have serious consequences for quality of your data.
+The effects of data corruption range from lower customer satisfaction to costly system reconciliation that results in service unavailability.
+
+{brandname} can carry out ACID (atomicity, consistency, isolation, durability) transactions to ensure the cache state is consistent.
+
+include::{topics}/locking.adoc[leveloffset=+1]
+include::{topics}/transactions.adoc[leveloffset=+1]
```

Comment:
@pruivo Does it make sense to combine locking and transactions here?

Reply:
Locks are not connected to transactions since we need the locks for non-transactions operations/caches too. 
I'll take a look tomorrow at the generated HTML to double check,